src/Project/ProjectVersionGit.php

```diff
@@ -36,6 +36,16 @@ public function getVersionString()
     }
 
     /**
+     * @return string closest tag of the project, read from git
+     */
+    public function getClosestTagString()
+    {
+        $version = $this->getGitData()['tag'];
+
+        return strstr($version, '-', true);
```

Comment:
We have release candidates, which include dashes as follows: 2.3-RC3 . If the above snipped handles these, then I think it's fine for me.